lib/template-parts.php

```diff
@@ -210,7 +210,7 @@ function filter_rest_wp_template_part_query( $args, $request ) {
 		);
 
 		// Ensure auto-drafts of all theme supplied template parts are created.
-		if ( wp_get_theme()->get( 'TextDomain' ) === $request['theme'] ) {
+		if ( wp_get_theme()->stylesheet === $request['theme'] ) {
```

Comment:
should we be use `wp_get_theme()->get_stylesheet()` instead? cc @mcsf

Reply:
@nosolosw dug into this just yesterday (and himself used `get_stylesheet()`), so I'll defer to him.

Reply:
:wave: TLDR: if you look for a theme identifier, `stylesheet` is the best option. Here's what I know:

- Despite its name, `stylesheet` refers to the name of the theme directory within the theme root (see https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/class-wp-theme.php#L206).
- For child themes, its value is the child theme directory name, not the parent's. This means that, effectively, `stylesheet` can function as the unique identifier of a theme, unlike other potential candidates such as `textdomain` (can be empty) or `template` (points to the parent in a child theme).
- If the theme is stored within another directory (eg: `themes/some-dir/my-theme` instead of `themes/my-theme`) it'll return `some-dir/my-theme`. If that's problematic you need to cover for that.

Reply:
Nice, thanks for that overview! I guess I was curious about the difference between:

```php
$theme_identifier = wp_get_theme()->stylesheet;
// alternatively:
$theme_identifier = wp_get_theme()->get_stylesheet();
```

is there any reason to use one over the other?

Reply:
Oh, what a misunderstanding, I'm so embarrassed :sweat_smile:

Probably the WordPress standards or PHP idioms have some strong opinions on how this should be done, but I don't know if it really matters in this case.

Reply:
I think `wp_get_theme()->get_stylesheet();` is preferred.